client/src/Pages/Uptime/Create/index.jsx

```
@@ -1,137 +1,203 @@
+	const triggerUpdate = () => {
+		setUpdateTrigger(!updateTrigger);
 	};
+
+	// Hooks
+	const [notifications, notificationsAreLoading, notificationsError] =
+		useGetNotificationsByTeamId();
+	const { determineState, statusColor } = useMonitorUtils();
+	// Network
 	const [isLoading] = useFetchMonitorById({
 		monitorId,
 		setMonitor: formatAndSet,
```

Biome (2.1.2)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)